brave-unbreak.txt

```diff
@@ -5,6 +5,13 @@
 ! pageview-api detection
 music.youtube.com##+js(brave-video-bg-play)
 weather.com,accuradio.com,mytuner-radio.com,awa.fm,radiko.jp,watch.jme.tv,play.asti.ga,app.hzp.co,boomplay.com,radio.net,live365.com,radioparadise.com,streema.com,radio.garden,yourclassical.org,audacy.com,internet-radio.com,di.fm,fmradiofree.com,beinsports.com,plus.nasa.gov,c-span.org,protonradio.com,ted.com,vimeo.com,radioonline.fm,jango.com,littlive.com,freefy.app,france.tv,rova.nz,radio-stations.co.nz,onlineradiobox.com,digitalradioplus.com.au,radio-australia.org,radioau.net,radio-uk.co.uk,liveradio.uk,ukonlineradio.com,liveradiouk.com,globalplayer.com,radioside.com,nebula.tv,britbox.com,freely.co.uk,bbc.co.uk,itv.com,dazn.com,nowtv.com,vudu.com,primevideo.com,pluto.tv,espn.com,xfinity.com,viaplay.com,rakuten.tv,joyn.de,canalplus.com,starplus.com,tubitv.com,dstv.com,directv.com,9now.com.au,10play.com.au,7plus.com.au,hayu.com,iview.abc.net.au,amcplus.com,neontv.co.nz,kick.com,docplay.com,acorn.tv,mubi.com,vix.com,nfl.com,foxsports.com,nbcsports.com,qobuz.com,iheart.com,music.amazon.com,liveone.com,tunein.com,tencentmusic.com,cnn.com,music.youtube.com,channel4.com,crunchyroll.com,fubo.tv,gem.cbc.ca,shudder.com,watch.att.com,crave.ca,tvnz.co.nz,threenow.co.nz,foxtel.com.au,stan.com.au,binge.com.au,skygo.co.nz,hulu.com,disneyplus.com,max.com,paramountplus.com,peacocktv.com,pandora.com,deezer.com,slingtv.com,discoveryplus.com,curiositystream.com,spotify.com,netflix.com,tidal.com,soundcloud.com,music.apple.com,nicovideo.jp,tiktok.com,twitch.tv,nzherald.co.nz##+js(brave-disable-pageview-api)
+
+!
+! Set trusted types scriptlet on Youtube
+! www.youtube.com##+js(set-trusted-types)
```

Comment:
Why is this commented out?

Reply:
- It wasn't doing anything we needed to change it too `www.youtube.com##+js(trusted_types)`
- Also Gorhill up'd changes making it https://github.com/gorhill/uBlock/commit/4f0d1301ab48e62afb3424609834c6353a0a5eee making the sciptlet not needed. Which we have sync'd. 
- Still working out what is actually broken, since the rules worked fine previously. Both trusted-rpnt sciptlets just broke